pkg/cluster/preinstall/tmpl/init_os.go

```diff
@@ -49,6 +49,9 @@ echo 'net.bridge.bridge-nf-call-ip6tables = 1' >> /etc/sysctl.conf
 echo 'net.bridge.bridge-nf-call-iptables = 1' >> /etc/sysctl.conf
 echo 'net.ipv4.ip_local_reserved_ports = 30000-32767' >> /etc/sysctl.conf
 
+#See https://imroc.io/posts/kubernetes/troubleshooting-with-kubernetes-network/
+echo “net.ipv4.tcp_tw_recycle = 0” >> /etc/sysctl.conf
```

Comment:
I see `tcp_tw_recycle` will be removed from the kernel after version 4.12.
Will there be a problem if all kernal version are added?

Reply:
Good suggestion. I've updated the code.